Sources/AblyAssetTrackingCore/Extension/Accuracy+CoreLocation.swift

```diff
@@ -0,0 +1,22 @@
+import CoreLocation
+
+public extension Accuracy {
+    func toCoreLocationAccuracy() -> CLLocationAccuracy {
+        switch self {
+        case .minimum:
+            if #available(iOS 14, *) {
+                return kCLLocationAccuracyReduced
+            } else {
+                return kCLLocationAccuracyThreeKilometers
+            }
+        case .low:
+            return kCLLocationAccuracyKilometer
+        case .balanced:
+            return kCLLocationAccuracyNearestTenMeters
```

Comment:
I think the amount of accuracy steps should match the one Apple gives us, so probably worth adding one more, f.e. `.sufficient` for kCLLocationAccuracyNearestTenMeters, and reduce `.balanced` to hundreds meters.

Reply:
This is a good idea but these enums are "untouchable" because it's designed some time ago to work on iOS, Android and many other platforms for AAT, so if I want to add new enum, every other platform should do the same moreover the documentation should be updated.
This PR is responsible just for mapping AAT Lib values to CoreLocation equivalents. If you don't feel alright with `kCLLocationAccuracyNearestTenMeters` for `.balanced` I would change it but please keep in mind there is no clear explanation why balanced should be mapped to `ten meters` or `hundred meters`  Or maybe there is? :)